src/content/developers/docs/consensus-mechanisms/pos/weak-subjectivity/index.md

```diff
@@ -0,0 +1,41 @@
+
+## Difference between weak subjectivity checkpoints and finalized blocks {#difference-between-ws-and-finalized-blocks}
+
+Finalized blocks and weak subjectivity checkpoints are treated differently by Ethereum nodes. If a node becomes aware of two competing finalized blocks then it is torn between the two - it has no way to identify automatically which is the canonical fork. This is symptomatic of a consensus failure. In contrast, a node simply rejects any block that conflicts with its weak subjectivity checkpoint. From the node's perspective the weak subjectivity checkpoint is represents an absolute truth that cannot be undermined by any new knowledge arriving from its peers.
+
+## How weak is weak? {#how-weak-is-weak}
+
+The subjective aspect of Ethereum's proof-of-stake is the requirement for a recent state (weak subjectivity checkpoint) from a trusted source to sync from. The risk of getting a bad weak subjectivity checkpoint is very low, partly because they can be checked against several independent public sources such as block explorers or multiple nodes. There is always some degree of trust required to run any software application, for example trusting that the software developers have produced honest software.
+
+A weak subjectivity checkpoint may even come as part of the client software. Arguably an attacker can corrupt the checkpoint in the software can just as easily corrupt the software itself. There is no real crypto-economic route around this problem, but the impact of untrustworthy developers is minimized in Ethereum by having multiple independent client teams each building equivalent software in different langages, all with a vested interest in maintaining an honest chain. Block explorers may also provide weak subjectivity checkpoints, or at least provide a way to cross-reference checkpoints obtained from elsewhere against an additional source.
+
+Finally, checkpoints can simply be requested from other nodes, perhaps another Etheruem user that runs a full node can provide a checkpoint that can then be verified against data from a block explorer. Overall, trusting the provider of a weak subjectivity checkpoint can be considered about as problematic as trusting the client developers. The overall trust required is low. It is also important to note that these considerations only become important in the very unlikely event where a majority of validators collude to produce an alternate fork of the blockchain. Under any other circumstances there is only one Ethereum chain to choose from.
```

Comment:
```suggestion
The subjective aspect of Ethereum's proof-of-stake is the requirement for a recent state (weak subjectivity checkpoint) from a trusted source to sync from. The risk of getting a bad weak subjectivity checkpoint is very low because they can be checked against several independent public sources such as block explorers or multiple nodes. However, there is always some degree of trust required to run any software application, for example, trusting that the software developers have produced honest software.

A weak subjectivity checkpoint may even come as part of the client software. Arguably an attacker can corrupt the checkpoint in the software and can just as easily corrupt the software itself. There is no real crypto-economic route around this problem, but the impact of untrustworthy developers is minimized in Ethereum by having multiple independent client teams, each building equivalent software in different languages, all with a vested interest in maintaining an honest chain. Block explorers may also provide weak subjectivity checkpoints or a way to cross-reference checkpoints obtained from elsewhere against an additional source.

Finally, checkpoints can be requested from other nodes; perhaps another Ethereum user that runs a full node can provide a checkpoint that validators can then verify against data from a block explorer. Overall, trusting the provider of a weak subjectivity checkpoint can be considered as problematic as trusting the client developers. The overall trust required is low. It is important to note that these considerations only become important in the very unlikely event that a majority of validators conspire to produce an alternate fork of the blockchain. Under any other circumstances, there is only one Ethereum chain to choose from.
```